plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/CachingHiveMetastore.java

```diff
@@ -495,36 +565,22 @@ private Optional<Table> loadTable(HiveTableName hiveTableName)
     @Override
     public PartitionStatistics getTableStatistics(Table table)
     {
-        HiveTableName tableName = hiveTableName(table.getDatabaseName(), table.getTableName());
-        PartitionStatistics statistics = get(tableStatisticsCache, tableName, () -> delegate.getTableStatistics(table));
```

Comment:
after the change, the `io.trino.plugin.hive.metastore.cache.CachingHiveMetastore#get(com.google.common.cache.Cache<K,V>, K, java.util.function.Supplier<V>)` becomes unused. remove.